migrations/sql/V2019.06.26.10.30__add_code_required_report_tables.sql

```diff
@@ -0,0 +1,173 @@
+CREATE TABLE mine_report_due_date_type (
```

Comment:
@klefler - I don't think we should tackle this on this PR. However, I see some inconsistencies with "type" tables vs "code" tables. I also see this propagated to the PK column.